frame/clearing-house/src/math.rs

```diff
@@ -0,0 +1,75 @@
+use crate::Config;
+use num_traits::{CheckedAdd, CheckedDiv, CheckedMul, CheckedSub};
+use sp_runtime::{
+	traits::Zero,
+	ArithmeticError,
+	ArithmeticError::{Overflow, Underflow},
+	FixedPointNumber,
+};
+
+pub fn decimal_abs_to_balance<T: Config>(decimal: &T::Decimal) -> T::Balance {
+	decimal
+		.saturating_abs()
+		.into_inner()
+		.try_into()
+		.map_err(|_| ArithmeticError::Underflow)
+		.expect("An absolute of Integer can always be converted to Balance")
+}
+
+pub fn decimal_checked_add<T: Config>(
+	a: &T::Decimal,
+	b: &T::Decimal,
+) -> Result<T::Decimal, ArithmeticError> {
+	// sign(a) sign(b) | CheckedAdd
+	// ----------------------------
+	//      -1      -1 | Underflow
+	//      -1       1 | Ok
+	//       1      -1 | Ok
+	//       1       1 | Overflow
+	a.checked_add(b).ok_or_else(|| match a.is_positive() {
+		true => Overflow,
+		false => Underflow,
```

Comment:
Just basic refactoring that I forgot to do. I imported `<variant>` directly to avoid specifying the whole enum path but forgot to apply it to all functions